cmd/tx.go

```diff
@@ -195,21 +197,21 @@ func createClientCmd(a *appState) *cobra.Command {
 				return err
 			}
 
-			c[src].PathEnd = path.End(c[src].ChainID())
-			c[dst].PathEnd = path.End(c[dst].ChainID())
+			src.PathEnd = path.End(src.ChainID())
+			dst.PathEnd = path.End(dst.ChainID())
 
 			// ensure that keys exist
-			if exists := c[src].ChainProvider.KeyExists(c[src].ChainProvider.Key()); !exists {
-				return fmt.Errorf("key %s not found on src chain %s", c[src].ChainProvider.Key(), c[src].ChainID())
+			if exists := src.ChainProvider.KeyExists(src.ChainProvider.Key()); !exists {
+				return fmt.Errorf("key %s not found on src chain %s", src.ChainProvider.Key(), src.ChainID())
```

Comment:
Maybe we want to change all the logging to utilize the chain name in a separate PR now that it is being used in most contexts instead of the chain ID?

Reply:
Agree, makes a lot of sense.